x/vesting/types/clawback_vesting_account.go

```diff
@@ -206,3 +206,13 @@ func (va ClawbackVestingAccount) UpdateDelegation(
 	va.DelegatedFree = newDelegated.Sub(va.DelegatedVesting)
 	return va, toClawBack
 }
+
+// HasLockedCoins returns true if the blocktime has not passed all clawback
+// account's lockup periods
+func (va ClawbackVestingAccount) HasLockedCoins(blockTime time.Time) bool {
+	unlockingTime := va.StartTime
+	for _, lp := range va.LockupPeriods {
+		unlockingTime.Add(time.Duration(lp.Length))
```

Comment:
do you need to set the value? or does it use a pointer?

```go
unlockingTime = unlockingTime.Add(time.Duration(lp.Length))
```

Reply:
good catch! Thank you!